internal/docgen/tpl.go

```diff
@@ -0,0 +1,69 @@
+package docgen
+
+const tplStr = `
+
+
+{{- define "namespace" -}}
+{{ template "do_not_edit" }}
+# Documentation for scw {{ .Cmd.Namespace }}
```

Comment:
```suggestion
# Documentation for `scw {{ .Cmd.Namespace }}`
```

Reply:
Looks like a bad idea as we cannot escape simple backtick in golang in raw litterals :(